docs/content/integrations/dbt/quickstart.mdx

```diff
@@ -90,25 +96,41 @@ defs = Definitions(
 </TabItem>
 <TabItem name="Option 2: Create a new Dagster project">
 
-You can create a Dagster project that wraps your dbt project by using the `dagster-dbt` command line interface.
+### Option 2: Create a new Dagster project
+
+<Note>
+  <strong>Heads up!</strong>: This approach uses the <code>DbtProject</code>{" "}
+  class, which is an experimental feature. Refer to the{" "}
+  <a href="/\_apidocs/libraries/dagster-dbt#dagster_dbt.DbtProject">
+    dagster-dbt API reference
+  </a>{" "}
+  for more information.
+</Note>
```

Comment:
This is true for Option 1 as well. 

Could we merge this PR without moving this note? Then in another PR, I can remove it completely and make the necessary changes to the Quickstart now that DbtProject is updated and no longer experimental.

Reply:
That's fine; I'll move it back!